### ISSUE TITLE
QMenu offset is not respected with context-menu

### ISSUE THREAD

**Issue by robfordww** [quasarframework/quasar#14944]
### What happened?

```
   <q-btn label="DEMO">
      <q-menu context-menu :offset='[-50, 20]'>
        <q-list dense style="min-width: 100px">
          <q-item clickable v-close-popup>
            <q-item-section>Close</q-item-section>
          </q-item>
```

`<q-menu context-menu :offset='[-50, 20]'>`  does not work , though the more regular `<q-menu :offset='[-50, 20]'> ` does work, but obviously this is not a context-menu.

### What did you expect to happen?

I expected it to offset the menu position.

### Reproduction URL

https://codepen.io/robfordww/pen/VwdyQdB

### How to reproduce?

just compare the two buttons and their menu position

### Flavour

Quasar CLI with Vite (@quasar/cli | @quasar/app-vite)

### Areas

Components (quasar)

### Platforms/Browsers

Chrome

**Comment by yusufkandemir** [comment]
@pdanpdan @rstoenescu if this behavior is intended, we should update the docs to state that the `offset` prop will not be respected when using `contextMenu` or `touchPosition` props. If it's not intended, then it's a bug.

**Comment by rstoenescu** [comment]
Hi,

The offset was never intended to work with touch-position or context-menu.
But added support for it now.

Will be available in Quasar v2.11.4
